modules/apps/translation/translation-api/src/main/java/com/liferay/translation/exporter/TranslationInfoFormValuesExporter.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.translation.exporter;
+
+import com.liferay.info.field.InfoFormValues;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import java.util.Locale;
+
+/**
+ * @author Alejandro Tardín
+ */
+public interface TranslationInfoFormValuesExporter<T> {
+
+	public InputStream export(
+			InfoFormValues infoFormValues, Locale sourceLocale,
```

Comment:
No thoughts on this?

Reply:
This one requires a bit more thought. 

Since `InfoFieldValue`s are linked to an `InfoField` I would say it depends on whether or not  `InfoField`s can exist independently from an `InfoForm`.

* If an `InfoField` is always part of an `InfoForm` I would say leave it as `InfoFormValues`. And maybe rename `InfoField` to `InfoFormField`.
* Otherwise, if there are other sources of `InfoField`s then it could make sense to give it a more generic name life `InfoFieldValues`.

Reply:
Makes sense to me.
I believe that InfoField could be used within a form, but it also makes sense to use it outside of one, which is why I decided to not call it InfoFormFields.

I believe that InfoFieldValues is a good name. I'll add a TODO to remember renaming it.